src/Scheduler.ts

```diff
@@ -17,7 +21,7 @@ import { Subscription } from './Subscription';
  *
  * @class Scheduler
  */
-export class Scheduler {
+export class Scheduler implements IScheduler {
 
   public static now: () => number = Date.now ? Date.now : () => +new Date();
```

Comment:
if `now` is defined in interface, shouldn't this be instance method as well to allow override?